approval_polls/static/approval_polls/create.js

```diff
@@ -1,33 +1,41 @@
-$(function () {
-  var that = this;
-  this.numChoiceFields = 4;
+$(function() {
+  var that = this; 
+  this.numChoiceFields = 4
+  this.currentNum = undefined;
```

Comment:
@Faxn What do you mean by data path? I added the currentNum fields to get the highest ID of the choice. With other changes I have now implemented, hope the use of currentNum makes sense.